src/main/java/com/github/shiranr/scooters/api/Admin.java

```diff
@@ -1,34 +1,45 @@
 package com.github.shiranr.scooters.api;
 
+import com.github.shiranr.scooters.db.Connection;
+import com.github.shiranr.scooters.db.CosmosClient;
+import com.github.shiranr.scooters.domain.Scooter;
+import com.github.shiranr.scooters.service.ScootersService;
+import com.github.shiranr.scooters.service.Service;
 import com.microsoft.azure.functions.*;
 import com.microsoft.azure.functions.annotation.AuthorizationLevel;
 import com.microsoft.azure.functions.annotation.BindingName;
 import com.microsoft.azure.functions.annotation.FunctionName;
 import com.microsoft.azure.functions.annotation.HttpTrigger;
 
 import java.util.Optional;
+import java.util.Random;
+import java.util.stream.IntStream;
 
 /**
  * Admin API - This section is restricted by permissions and should only be available to Admins.
  */
 public class Admin {
+
+    //I do not like this but for now I am not using spring so this is not injected.
+    Service service = new ScootersService(new CosmosClient(Connection.instance()));
```

Comment:
It's ok to create a technical debt, as long as you TODO it in a comment, (and usually adding a ticket number).
So Please add a TODO telling when it will be fixed.

Same for the rest of the files with this pattern